test/Microsoft.Dnx.DesignTimeHost.FunctionalTests/DthStartupTests.cs

```diff
@@ -38,23 +74,131 @@ public void DthStartup_GetProjectInformation(string flavor, string os, string ar
             {
                 client.Initialize(testProject);
 
-                var response = client.GetResponse<ProjectMessage>();
-                Assert.NotNull(response);
-                Assert.Equal(server.HostId, response.HostId);
-                Assert.Equal(0, response.ContextId);
-                Assert.Equal("ProjectInformation", response.MessageType);
+                var response = client.DrainTillFirst("ProjectInformation", timeout: TimeSpan.FromSeconds(10));
```

Comment:
It may be necessary given that the first initialization could be a bit slow. I can remove this for now, but we may need to come back to revisit.